Ginger/GingerCoreCommon/Repository/Environments/GeneralParam.cs

```diff
@@ -156,6 +156,53 @@ public static void UpdateNameChangeInItem(object item, string app, string prevVa
             }
         }
 
+        public static bool IsParamBeingUsedInBFs(object action, string appName, string paramOldName)
+        {
+            var actionMembers = action.GetType().GetMembers().Where(x => x.MemberType == MemberTypes.Property || x.MemberType == MemberTypes.Field);
+            foreach (MemberInfo mi in actionMembers)
+            {
+                if (Common.GeneralLib.General.IsFieldToAvoidInVeFieldSearch(mi.Name))
+                    continue;
+
+                dynamic value = GetActionMemberValue(action, mi);
+
+                if (value is IObservableList)
+                    foreach (object o in value)
+                        return IsParamBeingUsedInBFs(o, appName, paramOldName);
+                else
+                {
+                    try
+                    {
+                        if (value != null)
+                        {
+                            string stringValue = value.ToString();
+                            string placeHolder = "{EnvParam App=YYY Param=XXX}";
```

Comment:
Replace YYY and XXX here while creating the string to avoid replace call later.
Use string interpolation to maniulate the string